core/data_migration_scripts/EE_DMS_Core_4_1_0.dms.php

```diff
@@ -121,282 +121,282 @@ public function schema_changes_before_migration()
-					PAY_via_admin TINYINT(1) NOT NULL DEFAULT '0',
-					PAY_details TEXT COLLATE utf8_bin,
+        $sql = "PAY_ID int(10) unsigned NOT NULL AUTO_INCREMENT,
+					TXN_ID int(10) unsigned DEFAULT NULL,
+					STS_ID varchar(3) COLLATE utf8_bin DEFAULT NULL,
+					PAY_timestamp datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+					PAY_method varchar(45) COLLATE utf8_bin DEFAULT NULL,
+					PAY_amount decimal(10,3) DEFAULT NULL,
+					PAY_gateway varchar(32) COLLATE utf8_bin DEFAULT NULL,
+					PAY_gateway_response text COLLATE utf8_bin,
+					PAY_txn_id_chq_nmbr varchar(32) COLLATE utf8_bin DEFAULT NULL,
+					PAY_po_number varchar(32) COLLATE utf8_bin DEFAULT NULL,
+					PAY_extra_accntng varchar(45) COLLATE utf8_bin DEFAULT NULL,
+					PAY_via_admin tinyint(1) NOT NULL DEFAULT '0',
+					PAY_details text COLLATE utf8_bin,
 					PRIMARY KEY  (PAY_ID),
 					KEY TXN_ID (TXN_ID),
 					KEY PAY_timestamp (PAY_timestamp)";
         $this->_table_is_new_in_this_version($table_name, $sql, 'ENGINE=InnoDB ');
         $table_name = "esp_ticket";
-        $sql = "TKT_ID INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
-					  TTM_ID INT(10) UNSIGNED NOT NULL,
-					  TKT_name VARCHAR(245) NOT NULL DEFAULT '',
-					  TKT_description TEXT NOT NULL,
-					  TKT_qty MEDIUMINT(8) DEFAULT NULL,
-					  TKT_sold MEDIUMINT(8) NOT NULL DEFAULT 0,
-					  TKT_uses TINYINT NOT NULL DEFAULT '-1',
-					  TKT_min TINYINT UNSIGNED NOT NULL DEFAULT '0',
-					  TKT_max TINYINT NOT NULL DEFAULT '-1',
-					  TKT_price DECIMAL(10,3) NOT NULL DEFAULT '0.00',
-					  TKT_start_date DATETIME NOT NULL DEFAULT '0000-00-00 00:00:00',
-					  TKT_end_date DATETIME NOT NULL DEFAULT '0000-00-00 00:00:00',
-					  TKT_taxable TINYINT(1) UNSIGNED NOT NULL DEFAULT '0',
-					  TKT_order TINYINT(3) UNSIGNED NOT NULL DEFAULT '0',
-					  TKT_row TINYINT(3) UNSIGNED NOT NULL DEFAULT '0',
-					  TKT_is_default TINYINT(1) UNSIGNED NOT NULL DEFAULT '0',
-					  TKT_parent INT(10) UNSIGNED DEFAULT '0',
-					  TKT_deleted TINYINT(1) NOT NULL DEFAULT '0',
+        $sql = "TKT_ID int(10) unsigned NOT NULL AUTO_INCREMENT,
+					  TTM_ID int(10) unsigned NOT NULL,
+					  TKT_name varchar(245) NOT NULL DEFAULT '',
+					  TKT_description text NOT NULL,
+					  TKT_qty mediumint(8) DEFAULT NULL,
+					  TKT_sold mediumint(8) NOT NULL DEFAULT 0,
+					  TKT_uses tinyint(2) NOT NULL DEFAULT '-1',
+					  TKT_min tinyint(2) unsigned NOT NULL DEFAULT '0',
+					  TKT_max tinyint(2) NOT NULL DEFAULT '-1',
+					  TKT_price decimal(10,3) NOT NULL DEFAULT '0.00',
+					  TKT_start_date datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+					  TKT_end_date datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+					  TKT_taxable tinyint(1) unsigned NOT NULL DEFAULT '0',
+					  TKT_order tinyint(3) unsigned NOT NULL DEFAULT '0',
+					  TKT_row tinyint(3) unsigned NOT NULL DEFAULT '0',
+					  TKT_is_DEFAULT tinyint(1) unsigned NOT NULL DEFAULT '0',
```

Comment:
This doesn't look right here, should be `TKT_is_default` right?

Reply:
thanks